### PR TITLE
Add Adapted Rules.

### PR DESCRIPTION
**Pull request by JamesQuinley**
Adapted from https://sqlite.org/codeofethics.html, which we think fits our vision, and builds a common set of rules shared by most modern societies.

Signed-off-by: James Quinley <realgamer2023@hotmail.com>

### CODE REVIEW

**Comment by JamesQuinley** [comment]
Current file is TXT, considering making MD file but this works well for reading.

**Comment by asiekierka** [comment]
These aren't bad rules, but I'm worried about the following points and the ability of the PolyMC project to adequately enforce them:

- `First of all, love the Lord God with your whole heart, your whole soul, and your whole strength.` - Minecraft as a game explicitly promotes un-Christian values, including elements of the occult and ~~subversive leftist propaganda~~.
- `Do not become attached to pleasures.` - PolyMC is a video game launcher project. It serves the explicit purpose of lowering the barrier of accessibilty of both the development and proliferation of video games, which leads a lot of players into addiction.
- `Speak no useless words or words that move to laughter.` - While I am willing to point out that the Discord server has removed the ability for lurkers to be negatively impacted by said "useless words", I am still concerned that the lead developers are engaging in or otherwise condoning acts of playful trolling outside of public view.
- `Hate no one.` - this attempt at ~~subversive leftist propaganda~~ cannot stand in a project like PolyMC, naturally.

Most of the other rules seem fine in the context of what PolyMC is trying to be.

In other words, this document is about as useful as a Code of Conduct is to the average open source project: a neat-sounding value system copy-pasted from another project which will ultimately be ignored once someone's real-world interests get threatened in practice. Disappointing - I'd like to see more documents which accurately reflect the values the community stands by, rather than ones which sound good on paper.

**Review by LennyMcLennington — APPROVED** [review]
lol

it might be nice to have it as a markdown document, it would probably be a bit nicer to read. The only thing that really needs changing to convert it to markdown is adding the headings with #

**Comment by ghost** [comment]
lol

**Comment by JamesQuinley** [comment]
> lol
> 
> it might be nice to have it as a markdown document, it would probably be a bit nicer to read. The only thing that really needs changing to convert it to markdown is adding the headings with #

This feedback was noted and I've made needed changes.
-James

**Review by Sneedplex — APPROVED** [review]
LGTM.

File: Rules.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
-1. Purpose
+# 1. Purpose
 The Founder of PolyMC, and all of the current developers at the time when this document was composed, have pledged to govern their interactions with each other, 
 with their clients, and with the larger PolyMC user community in accordance with the "instruments of good works" from chapter 4 of The Rule of St. Benedict (hereafter: "The Rule"). 
 This code of ethics has proven its mettle in thousands of diverse communities for over 1,500 years, and has served as a baseline for many civil law codes since the time of Charlemagne. 
 
-1.1. Scope of Application
+# 1.1. Scope of Application
```
Comment (slendidev) [inline Rules.md:6]: If it's "1.1", shouldn't it be a "##" instead of "#"?

**Review by slendidev — APPROVED** [review]
LGTM.

**Review by ghost — APPROVED** [review]

**Comment by JamesQuinley** [comment]
Forgot to close, this was a joke.
